src/fu-security-attr.c

```diff
@@ -174,15 +174,30 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
 		return g_strdup(_("fwupd plugins"));
 	}
 	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_DCI_ENABLED) == 0 ||
-	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_DCI_LOCKED) == 0) {
-		/* TRANSLATORS: Title: Direct Connect Interface (DCI) allows
-		 * debugging of Intel processors using the USB3 port */
-		return g_strdup(_("Intel DCI debugger"));
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_DEBUG_LOCKED) == 0) {
+		/* TRANSLATORS: Title: Allows debugging of parts using proprietary hardware */
+		return g_strdup(_("Debugging"));
 	}
```

Comment:
Platform Debugging? Or CPU Debugging? The sole word is a bit generic...

Reply:
It's platform. Very rarely would you be debugging CPU cores directly. Will update.

Reply:
Isn't DCI exactly that? i.e. CPU-level debugging?

Reply:
IMO CPU debugging is a side effect; aren't you getting access to platform registers from it and that's usually where the focus is?

Reply:
Taking the Intel view - DCI is CPU debugging **and** PCH debugging.  So I think "Platform" debugging is appropriate for that as well as AMD.  I feel it could in the future cover things like JTAG on ARM too.

Reply:
I think it makes sense if DCI, the AMD on and JTAG are all the same HSI level -- which maybe they are?

Reply:
yeah they are IMO.  My take is we should combine this attribute for Intel and AMD and then if ARM ends up being different we cross that bridge at that time.

Reply:
> DCI is CPU debugging and PCH debugging

Really? I've not seen people debug the PCH using DCI, although I've not really played with it myself.

Reply:
https://conference.hitb.org/hitbsecconf2017ams/materials/D2T4%20-%20Maxim%20Goryachy%20and%20Mark%20Ermalov%20-%20Intel%20DCI%20Secrets.pdf shows that you can get at everything.  It's not just CPUs.

Reply:
Okay, agreed, thanks for the link.

Reply:
OK, in that case I'm going to rename the other DCI attribute as well to match and I'll drop a link to this in the description too.